Sources/MarkdownBabel/ExecutableContext/ImageResult.swift

```diff
@@ -6,12 +6,15 @@ public struct ImageResult: ResultMarkup {
 
 	public let title: String
 	public let source: String
+	public let alt: String
+	public var contentHash: String { alt }
```

Comment:
We now have 2 places that know that the content hash is stored in the alt text part of the image. There's potential to make this 1 value type.